bluesky/simple_scans.py

```diff
@@ -66,8 +67,8 @@ class _BundledScan:
 
     def __init__(self):
         # subs and sub_factories can be set individually per instance
-        self.subs = self.default_subs
-        self.sub_factories = self.default_sub_factories
+        self.subs = list(self.default_subs)
```

Comment:
I really don't like that these things can be dicts or lists.  We should to settle on all subscriptions need to be able to take everything (and gracefully ignore those they can't handle) and this is a list or you must always be specific about these being dicts.